### ISSUE TITLE
An unhandled exception has occurred. See browser dev tools for details. Reload

### ISSUE THREAD

**Issue by luizpires** [ADefWebserver/SyncfusionHelpDesk#4]
Hi, 

I am currently reading the Blazor Succintly eBook and found some problems when trying to run EditTicket control (page 73).
Considering the possibility that I had made a mistake when following the construction of the application, I downloaded the source code from GitHub and got the same error, see below:

![image](https://user-images.githubusercontent.com/7466593/87341108-6a854b80-c51f-11ea-819c-663f371f5d10.png)

Best Regards,

Luiz Pires

**Comment by ADefWebserver** [comment]
What version of Syncfusion and .Net Core are you using?
Also, what are the exact steps you did that produced the error?

Thank You

**Comment by luizpires** [comment]
Hi, 
Syncfusion.Blazor version 18.2.0.44 and .Net Core 3.1

The error occurs immediately after clicking the Edit button. The form freezes.

Thank You

**Comment by ADefWebserver** [comment]
I wrote this for a different version of Syncfusion (and it was tested, I promise! :))
Please open a help desk ticket with Syncfusion because they could have introduced a bug.

Thanks!

**Comment by luizpires** [comment]
Ok, 

Thank You

**Comment by ADefWebserver** [comment]
If they need me to change my code (I shouldn't!), please let me know. Thank You

**Comment by ADefWebserver** [comment]
(reopening this for now...)

**Comment by luizpires** [comment]
Hello Michael,
You do not need to make changes to the SyncHelpDesk Application code, see below for the solution to the question answered by Syncfusion support.

"We have validated the reported issue and the reported issue occurs while placing the GridCommandColumns at the first position. Since it is a known issue we have resolved the issue and it was included in our latest patch release 18.2.0.45. So kindly upgrade to our latest version to resolve the reported issue."  

Thank you very much for your attention and support.

Luiz Pires
